stylesheets/components/_card.scss

```diff
@@ -0,0 +1,137 @@
+// -----------------------------------------------------------------------------
+// This file contains all styles related to the card component.
+// -----------------------------------------------------------------------------
+
+/**
+* Mixin for text used in cards
+**/
+@mixin card-text {
+  color: $night-grey;
+  font-family: $sans-serif-stack;
+  font-size: 15px;
+  font-weight: $font-weight-normal;
+  margin-top: 8px;
+
+  @include lg-up {
+    margin-top: 10px;
+  }
+}
+
+/**
+* Mixin for title text used in cards
+**/
+@mixin card-title-text {
+  @include card-text;
+
+  font-size: 18px;
+  font-weight: $font-weight-bold;
+  margin-top: 0;
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    color: $night-grey;
+    text-decoration: none;
+  }
+}
+
+/**
+* Mixin for flex styles used in all card elements
+**/
+@mixin card-element-flex {
+  display: flex;
+  flex-wrap: wrap;
+}
```

Comment:
I can see this as a candidate for a global mixin.

Reply:
Agree - I think we've used flexbox really inconsistently across (and even within) our web properties and we could probably make our lives a lot easier if we started using it more consistently. To me this kind of gets into the world of layout stuff, so as you're suggesting it might make sense to hold off on this until we are a little further down the road.